src/app/lib/config/services/news.js

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
Can you extract these out to variables? If they are exported default in such a way, they will be named `default` in a stack-trace, which is harder to debug.